core/src/edu/asu/c3simulator/screens/FundingScreen.java

```diff
@@ -49,29 +50,27 @@ public void clicked(InputEvent event, float x, float y)
 				- (0.01f * stage.getWidth()), (0.01f * stage.getWidth()));
 		
 		// TODO add Corner Advisor
-		// TODO add Home Button
+		HomeButton home = new HomeButton(game);
 		
-		NavigationPanel navigationPanel = createNavigationPanel();
-		
-		stage.addActor(navigationPanel);
+		stage.addActor(home);
 		stage.addActor(continueButton);
 		stage.addActor(ScreenDiscription);
 	}
 	
 	@Override
-	public NavigationPanel createNavigationPanel()
+	public void createNavigationPanel()
```

Comment:
The purpose of having this method is to create all navigation panels after all screens are created and referenced. However, while working with Justin on linking the screens, I noticed that I was still creating them in the constructor of each screen. So, we changed to make the methods create and add the panel to their screen